ibis/common/annotations.py

```diff
@@ -89,6 +96,17 @@ def optional(cls, validator=None, default=None):
             validator = option(validator, default=default)
         return cls(validator, default=None)
 
+    @classmethod
+    def varargs(cls, validator=None):
```

Comment:
`variadic` could be a slightly better naming choice

Reply:
I think `varargs` is fine, if only for consistency with `*args, **kwargs`.

Reply:
Ok, keeping as is.